packages/libs/client-examples/README.md

```diff
@@ -1,17 +1,155 @@
+): Promise<void> {
+  // The first step for interacting with the blockchain is creating a request object.
+  // We call this object a transaction object. `Pact.builder` will help you to create this object easily.
+  const transaction = Pact.builder
+    // In Pact, we have two types of commands: execution and continuation. Most of the typical use cases only use execution.
+    .execution(
+      // This function uses the type definition we generated in the previous step and returns the pact code as a string.
+      // This means you want to call the transfer function of the coin module (contract).
+      // You can open the coin.d.ts file and see all of the available functions.
+      Pact.modules.coin.transfer(
+        sender.accountName,
+        receiver.accountName,
+        // As we know JS rounds float numbers, which is not a desirable behavior when you are working with money. So instead, we send the amount as a string in this format.
+        {
+          decimal: amount,
+        },
+      ),
+    )
+    // The sender needs to sign the command; otherwise, the blockchain node will refuse to do the transaction.
+    .addSigner(sender.publicKey, (withCapability) => [
+      // The sender also mentions they want to pay the transaction fee by adding the "coin.GAS" capability.
+      withCapability('coin.GAS'),
+      // The sender scopes their signature only to "coin.TRANSFER" with the specific arguments.
+      withCapability(
+        'coin.TRANSFER',
+        sender.accountName,
+        receiver.accountName,
+        {
+          decimal: amount,
+        },
+      ),
+    ])
+    // Since Kadena has a multi-chain architecture, we need to set the chainId.
+    // We also need to mention who is going to pay the gas fee.
+    .setMeta({ chainId: '0', sender: sender.accountName })
+    // We set the networkId to "testnet04"; this could also be "mainnet01" or something else if you use a private network or a fork.
+    .setNetworkId('testnet04')
+    // Finalize the command and add default values and hash to it. After this step, no one can change the command.
+    .createTransaction();
+
+  // The transaction now has three properties:
+  // - cmd: stringified version of the command
+  // - hash: the hash of the cmd field
+  // - sigs: an array that has the same length as signers in the command but all filled by undefined
+
+  // Now you need to sign the transaction; you can do it in a way that suits you.
+  // We exported some helpers like `signWithChainweaver` and signWithWalletConnect, but you can also use other wallets.
+  // For example, if you are using EckoWallet, it has a specific API for signing transactions.
+  // By the end, the signer function should fill the sigs array and return the signed transaction.
+  const signedTr = await signWithChainweaver(transaction);
+
+  // As the signer function could be an external function, we double-check if the transaction is signed correctly.
+  if (isSignedCommand(signedTr)) {
```

Comment:
Even though it's example code, I'd still recommend inverting the if statement to get early returns instead.

Reply:
```
if (!isSignedCommand(signedTr)) throw new Error('Transaction has not been signed');

... other code ...
if (response.result.status === 'failure) throw response.result.error;

console.log('yaaaaay')
```

Reply:
Done!